### ISSUE TITLE
[BUG] Crash when opening a new tab from UI

### ISSUE THREAD

**Issue by jperl** [microsoft/playwright#4472]
**Context:**
- Playwright Version: 1.6.2
- Operating System: Mac 11.0.1 Big Sur, Ubuntu Bionic
- Node.js version: 12.18.3
- Browser: Chromium

**Code Snippet**

```javascript
const playwright = require("playwright");

(async () => {
  const browser = await playwright.chromium.launch({ headless: false });

  const context = await browser.newContext();
  const page = await context.newPage();
  await page.goto("https://google.com/");

  // open a new tab from the UI in chrome, process immediately exits
})();
```

**Describe the bug**

Opening a tab in headed mode closes the process, no error thrown.

**Comment by jperl** [comment]
On ubuntu bionic 1.4.0 works, but 1.5.0 and 1.5.2 I get this error:

```
/node_modules/playwright/lib/client/connection.js:138
            throw new Error(`Cannot find parent object ${parentGuid} to create ${guid}`);
            ^

Error: Cannot find parent object BrowserContext@cd1cd86f076f4a2719ebd1d3dd1a42cb to create Frame@61a33d06ce441dac1aa262f34707d3ec
    at Connection._createRemoteObject (/node_modules/playwright/lib/client/connection.js:138:19)
    at Connection.dispatch (/node_modules/playwright/lib/client/connection.js:105:18)
    at Immediate.<anonymous> (/node_modules/playwright/lib/inprocess.js:40:85)
    at processImmediate (internal/timers.js:461:21)
```

I cannot test earlier versions on my mac since I updated to Big Sur.

**Comment by dgozman** [comment]
[Upstream issue](https://bugs.chromium.org/p/chromium/issues/detail?id=1149654).

**Comment by dgozman** [comment]
Fixed by #4626.
